.github/workflows/main.yaml

```diff
@@ -349,3 +351,57 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: [ 'cri-o', 'docker' ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
```

Comment:
It would be better to install dependences include ginkgo, kind and kubectl to keep consistent with e2e test, otherwise it may occur some warning like: 
![image](https://github.com/kubeedge/kubeedge/assets/48788150/e25d6665-8d85-4624-a238-9d190fb6b2d5)

Reply:
ok, I have installed.